pkg/controller/process.go

```diff
@@ -33,17 +33,21 @@ func (c AppGwIngressController) Process(event events.Event) error {
 		return errors.New("unable to get specified ApplicationGateway")
 	}
 
+	envVars := environment.GetEnv()
+	prohibitedTargets := c.k8sContext.ListAzureProhibitedTargets()
+
 	cbCtx := &appgw.ConfigBuilderContext{
-		ServiceList:          c.k8sContext.ListServices(),
-		IngressList:          c.k8sContext.ListHTTPIngresses(),
-		ProhibitedTargets:    c.k8sContext.ListAzureProhibitedTargets(),
-		IstioGateways:        c.k8sContext.ListIstioGateways(),
-		IstioVirtualServices: c.k8sContext.ListIstioVirtualServices(),
-		EnvVariables:         environment.GetEnv(),
+		ServiceList:                c.k8sContext.ListServices(),
+		IngressList:                c.k8sContext.ListHTTPIngresses(),
+		ProhibitedTargets:          c.k8sContext.ListAzureProhibitedTargets(),
+		IstioGateways:              c.k8sContext.ListIstioGateways(),
+		IstioVirtualServices:       c.k8sContext.ListIstioVirtualServices(),
+		EnvVariables:               environment.GetEnv(),
```

Comment:
minor thing, but you could change this to EnvVariables: envVars since you already called the GetEnv() function above

Reply:
Yes! Thank you!